proto/message_contents/invitation.proto

```diff
@@ -70,3 +73,14 @@ message SealedInvitation {
     SealedInvitationV1 v1 = 1;
   }
 }
+
+// Payload for user's consent proof to be set in the invitation
+// Signifying the conversation should be preapproved for the user on receipt
+message ConsentProofPayload {
+  // the user's signature in hex format
+  string signature = 1;
+  // approximate time when the user signed
+  uint64 timestamp = 2;
+  // version of the payload
+  uint32 payload_version = 3;
```

Comment:
I don't have a strong preference here, but it might be nicer to have an Enum for the payload version. That's what we do in other places, even if the only option right now is `VERSION_ONE`